commonv5/ts/sonarqube/Analysis.ts

```diff
@@ -44,145 +38,124 @@ const separatorStyle = `
   margin: 16px 0px;`;
 
 export default class Analysis {
-  constructor(
-    private readonly analysis: IAnalysis,
-    private readonly endpointType: EndpointType,
-    private readonly warnings: string[],
-    private readonly dashboardUrl?: string,
-    private readonly metrics?: Metrics,
-    private readonly projectName?: string,
```

Comment:
Consolidated all these constructor params into a `result` object (was a SQ code smell to have too many constructor parameters)